tutorials/high-performance-apps.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: 'Build High-Performance Applications'
+description: 'The world’s largest distributed GPU cloud at the most competitive prices'
+---
+
+This tutorial introduces the distributed and dynamic nature of SaladCloud, highlights common challenges when migrating
+workloads from hyperscalers to SaladCloud, and shares best practices along with proven insights from customers who have
+successfully built large-scale AI inference applications and run molecular dynamics simulations, using tens to thousands
+of Salad nodes.
```

Comment:
I'm unclear if this should be Salad nodes or SaladCloud nodes. I'd probably lean the latter but whatever is consistent